browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -40,6 +40,64 @@ test.describe('program creation', () => {
     await adminProgramImage.expectProgramImagePage()
   })
 
+  test('create program with disabled visibility condition feature enabled', async () => {
+    const {page, adminPrograms, adminProgramImage} = ctx
+    await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
+    await loginAsAdmin(page)
+
+    await adminPrograms.addProgram(
+      'program name',
+      'description',
+      'https://usa.gov',
+      ProgramVisibility.DISABLED,
+      'admin description',
+      /* isCommonIntake= */ false,
+      'selectedTI',
+      'confirmationMessage',
+      Eligibility.IS_GATING,
+      /* submitNewProgram= */ false,
+    )
+    await adminPrograms.expectProgramDetailsSaveAndContinueButton()
+    await validateScreenshot(
```

Comment:
suggestion (non-blocking): you could also test that `Disabled` text shows up on the page.

Reply:
done!